regulations/static/regulations/css/less/variables.less

```diff
@@ -33,6 +33,8 @@ variables.less contains all theme variable / variable overrides
 @red_orange_80: #D96A4F;
 @red_orange_50: #E9A193;
 
+@history_drawer_border_color: #d1d3d5;
```

Comment:
This seems awfully specific. Is there a more generic name that we could use to define the _pallet_? Ideally, a new site should be a matter of saying "these are the primary color, these are the accents, etc." rather than targeting specific elements.
Also, it looks like the standard set above is to use caps when defining color hex.